docs/openapi/components/schemas/credentials/BankAccountCredential.yml

```diff
@@ -124,19 +118,5 @@ example: |-
         "addressCountry": "Canada",
         "postalCode": "M3B 1A2"
       }
-    },
-    "credentialStatus": {
-      "id": "https://example.com/status/362bbbb5-3d9b-4405-82ee-cf1443415351#5",
-      "type": "StatusList2021Entry",
-      "statusPurpose": "revocation",
-      "statusListIndex": "5",
-      "statusListCredential": "https://example.com/status/362bbbb5-3d9b-4405-82ee-cf1443415351"
-    },
-    "proof": {
```

Comment:
The updated values here were 
```
    "proof": {
      "type": "Ed25519Signature2020",
      "created": "2023-06-20T19:35:30Z",
      "verificationMethod": "urn:uuid:11111111-1111-1111-1111-111111111111#key-1",
      "proofPurpose": "assertionMethod",
      "proofValue": "z4xTXcWHhZY8oXCXTKS...cSKGK9YcBuKqyqzjbaohmtMZBAenC9huBJ"
    }
```

Reply:
yes, this is prep to support v2 data model

Reply:
I saw that draft- looking forward to learning about the changes.  In terms of this PR, is there anything outstanding @mkhraisha ?

Reply:
No that tackles them all thank you!